addons/flannel/template/generate.sh

```diff
@@ -10,10 +10,14 @@ function generate() {
     cp -r ./base/* "$dir"
 
     download_yaml
-
+    replace_flannel_image_with_version
     find_images_in_yaml
 }
 
+function replace_flannel_image_with_version() {
+    sed -i -e "s/flannel:v0.20.2/flannel:v$version/" "$dir/yaml/kube-flannel.yml"
```

Comment:
Unfortunately this one always worked because they will be updating the version in the docs soon

We could probably use a regex instead

Reply:
@laverya , thank you for catching that. 

OH, that is a mistake I pushed my test. 
It is within the regex of course. 
Hold on.